x-pack/plugins/data_visualizer/public/application/file_data_visualizer/components/file_data_visualizer_view/file_error_callouts.tsx

```diff
@@ -146,3 +146,26 @@ export const Explanation: FC<{ error: FindFileStructureErrorResponse }> = ({ err
     </>
   );
 };
+
+export const FindFileStructurePermissionDenied: FC = () => {
+  return (
+    <>
+      <EuiCallOut
+        title={
+          <FormattedMessage
+            id="xpack.dataVisualizer.file.fileErrorCallouts.findFileStructurePermissionDenied.title"
+            defaultMessage="Permission denied"
+          />
+        }
+        color="danger"
+        iconType="cross"
+        data-test-subj="dataVisualizerFileStructurePermissionDeniedErrorCallout"
+      >
+        <FormattedMessage
+          id="xpack.dataVisualizer.file.fileErrorCallouts.findFileStructurePermissionDenied.description"
+          defaultMessage="You must have monitor_text_structure or monitor cluster privileges to use this feature."
```

Comment:
Would it be useful to explain why monitor_text_structure or monitor permissions are needed? Maybe linking to https://www.elastic.co/guide/en/elasticsearch/reference/current/find-structure.html for permission details?